backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1057,6 +1057,10 @@ class FileInfoAbstractSchema(marshmallow.Schema):
         description="Is pdf version of file available ?",
         example=True,
     )
+    jpeg_available = marshmallow.fields.Bool(
+        description="Is jpeg version of file available ?",
```

Comment:
=> "true if a JPEG preview is available or false"

Reply:
done